lib/app/features/user/providers/tab_data_source_family.c.dart

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:ion/app/features/nostr/providers/entities_paged_data_provider.c.dart';
+import 'package:ion/app/features/user/model/tab_entity_type.dart';
+import 'package:ion/app/features/user/providers/user_articles_data_source_provider.c.dart';
+import 'package:ion/app/features/user/providers/user_posts_data_source_provider.c.dart';
+import 'package:ion/app/features/user/providers/user_replies_data_source_provider.c.dart';
+import 'package:ion/app/features/user/providers/user_videos_data_source_provider.c.dart';
+
+final tabDataSourceProviderFamily =
```

Comment:
Let's change this provider on the generated one to align with one style for the providers used in the project

Reply:
Done.